submissions/innazozulia/Hooli-style-popup/style.css

```
@@ -0,0 +1,143 @@
+
+body, h1 {
+    margin: 0;
+}
+
+h1{
+padding-left:15px;
+}
+.container {
+    width: 1200px;
+    max-width: 100%;
+    padding: 0 15px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+.nav-list{
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  padding: 18px;
+  padding-bottom: 14px;
+  margin-right: 30px;
+  margin-left: 30px;
+  background-color: rgba(25, 114, 136, 0.067);
+}
+.nav-list .nav-item {
+    font-size: 16px;
+    margin-left: 40px;
+    position: relative;
+    cursor: pointer;
+}
+.nav-list .nav-item label {
+    cursor: pointer;
+}
+.logo{
+    position:absolute;   
+    padding-left: 50px;
+} 
+.nav-list .nav-item .nav-link {
+    display: inline-block;
+    text-decoration: none;
+    color: #000;
+    list-style-type: none;   
+}
+
+.nav-list .nav-item .nav-link:focus {
+    outline: 1px solid #cdcdcd;
+}
+.nav-list .nav-item .nav_icon {
+    font-size: 22px;
+}
+.nav-list .nav-item .submenu-title {
+    margin-top: 5px;
+}
+.nav-list .submenu {
+    display: none;
+    position: absolute;
+    width: 270px;
+    max-height: 260px;
+    top: 40px;
+    right: -50px;
+    padding: 20px 25px 10px;
+    background: #fff;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+    overflow-y: auto;
+}
+.nav-list .hidden-checkbox {
+    position: absolute;
+    width: 0;
+    height: 0;
+    opacity: 0;
+    cursor: pointer;
+}
+.nav-list .hidden-checkbox:focus + label {
+    outline: 1px solid #cdcdcd;
+}
+.nav-list .submenu-checkbox:checked ~ .submenu {
+    display: block;
+}
+.nav-list .submenu-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    width: 100%;
+    padding: 0;
+    list-style: none;
+}
+.nav-list .submenu-list .submenu-item {
```

No need to build such a complex selector. I think it would be enough to use just selector for the `.submenu-item`
```suggestion
 .submenu-item {
```